src/devices/bus/sg1000_exp/sp400.h

```diff
@@ -0,0 +1,62 @@
+#ifndef MAME_BUS_SG1000_EXP_SK1100_SP400_H
```

Comment:
As above, the licensing and copyright holder headers are entirely missing on this file.

Reply:
thank you, fixed